grails-app/assets/javascripts/streama/directives/streama-video-player-directive.js

```diff
@@ -101,6 +101,13 @@ angular.module('streama').directive('streamaVideoPlayer', [
               var selectedVideoFile = _.find($scope.options.videoFiles, {label: savedVideoFileLabel});
               changeVideoFile(selectedVideoFile);
             }
+            //Autoloads sub#0 if the setting is true on Admin>Settings, and if the subtitle exists (by Norwelian)
+            apiService.settings.list().then(function(data){
+                if(data['data'][18]['parsedValue'] && $scope.options.subtitles){
```

Comment:
data['data'][18]? the must be a better way to get to the value by name, no? using "subtitle_auto_load"

Reply:
I 100% agree.
The way is done now is way safer, and it defaults to false, so if the setting does not exist, it simply won't activate.